README.md

```diff
@@ -21,10 +21,15 @@
 
 ## Development
 
-See [DEVELOPMENT](DEVELOPMENT.md) for information about code style,
+See [DEVELOPMENT](.github/DEVELOPMENT.md) for information about code style,
 development process, and guidelines.
 
-See [CONTRIBUTING](CONTRIBUTING.md) for contribution requirements.
+See [CONTRIBUTING](.github/CONTRIBUTING.md) for contribution requirements.
```

Comment:
Maybe both should be in `docs` -- also a Github-allowed location, and sounds more human-oriented.

Reply:
I thought about that, but docs is the source tree for end user documentation, so putting those in there seems confusing or wrong. However, I don’t feel strongly, so open to others opinions here.

Reply:
The `docs` folder is the natural place people would look, so it makes sense from that standpoint. However, since it’s the documentation source module, and we already have a `README` there about just the documentation, I’m concerned people might see `DEVELOPMENT` or `CONTRIBUTING` as applying specifically to the documentation module. But my viewpoint is obviously biased and I might be over indexing on seeing that as a source code module.

There’s also the fact that since it is a source code module, putting project level documentation there seems wrong.